on-prem/installation/installation_guide.md

```diff
@@ -4,28 +4,19 @@
 
 Stoplight consists of three container services:
 
-- The frontend app
-- The backend API
+- Stoplight Frontend
+- Stoplight Backend
```

Comment:
I think these should be capitalized as the proper names of our applications, like Oracle Java or Apache Kafka

Reply:
```suggestion
- Stoplight backend
```

Reply:
Java and Kafka are registered trademarks, not to mention things in their own right (a programming language and a software platform). Has Stoplight registered "Stoplight Backend" and "Stoplight Frontend", or are we just referring to the frontend and backend of the application here? Neither of them are standalone applications or frameworks.

Reply:
These are the names of our application components, not descriptions. Our other components include Studio, Prism, Pubs, etc. We wouldn't call it "Stoplight studio" just because "studio" is a common word--it's the name of our product.

Reply:
This "felt wrong" so I double checked with Taylor about Stoplight's branding. The app backend and frontend are not branded components like Studio, Prism etc. Definitely lowercase here.